base/expr.jl

```diff
@@ -118,6 +118,22 @@ macro eval(x)
     :($(esc(:eval))($(Expr(:quote,x))))
 end
 
+"""
+    @inline
+
+Give a hint to the compiler that this function is worth inlining.
+
+Small functions typically don't even need the @inline annotation,
```

Comment:
Should wrap `@inline` in backticks like you've done else where in the docstring.